test/pages/uat_actions/generate_test_run.spec.js

```diff
@@ -0,0 +1,186 @@
+        localVue,
+        data() {
+          return {
+            plateSpecs: [{ numberOfPlates: 2, numberOfPositives: 1 }, { numberOfPlates: 11, numbeOfPpositives: 3 }],
+          }
+        },
+      })
+
+      expect(wrapper.vm.totalPlates).toEqual(2 + 11)
+    })
+  })
+
+  // methods
+  describe('#numberOfPositivesOptions', () => {
+    it('has a default', () => {
+      expect(wrapper.vm.numberOfPositivesOptions()).toHaveLength(97)
+    })
+  })
+
+  describe('#numberOfPlatesOptions', () => {
+    it('has a default', () => {
+      expect(wrapper.vm.numberOfPlatesOptions()).toHaveLength(200)
+    })
+  })
+
+  describe('#add', () => {
+    it('when plateSpecs is an empty list', async () => {
+      wrapper = mount(GenerateTestRun, {
+        localVue,
+        data() {
+          return {
+            form: { numberOfPlates: 1, numberOfPositives: 2 },
+          }
+        },
+      })
+
+      await wrapper.find('#addButton').trigger('click')
+
+      expect(wrapper.vm.plateSpecs).toEqual([{ numberOfPlates: 1, numberOfPositives: 2 }])
+      expect(wrapper.vm.form.numberOfPlates).toEqual(1)
+      expect(wrapper.vm.form.numberOfPositives).toEqual(0)
+    })
+
+    it('when plateSpecs is not empty', async () => {
+      wrapper = mount(GenerateTestRun, {
+        localVue,
+        data() {
+          return {
+            plateSpecs: [{ numberOfPlates: 1, numberOfPositives: 2 }],
+            form: { numberOfPlates: 3, numberOfPositives: 4 },
+          }
+        },
+      })
+
+      await wrapper.find('#addButton').trigger('click')
+
+      expect(wrapper.vm.plateSpecs).toEqual([{ numberOfPlates: 1, numberOfPositives: 2 }, { numberOfPlates: 3, numberOfPositives: 4 }])
+      expect(wrapper.vm.form.numberOfPlates).toEqual(1)
+      expect(wrapper.vm.form.numberOfPositives).toEqual(0)
+    })
+  })
+
+  describe('#reset', () => {
+    it('resets plateSpecs', async () => {
+      wrapper = mount(GenerateTestRun, {
+        localVue,
+        data() {
+          return {
+            plateSpecs: [{ numberOfPlates: 1, numberOfPositives: 2 }],
+          }
+        },
+      })
+
+      await wrapper.find('#resetButton').trigger('click')
+
+      expect(wrapper.vm.plateSpecs).toEqual([])
+    })
+  })
+
+  describe('#showAlert', () => {
+    it('calls alert show', () => {
+      wrapper.vm.$refs.alert.show = jest.fn()
+      wrapper.vm.showAlert('message', 'success')
+      expect(wrapper.vm.$refs.alert.show).toHaveBeenCalled()
+    })
+  })
+
+  describe('#generateTestRun', () => {
+    let wrapper
+
+    beforeEach(() => {
+      const $router = {
+        push: jest.fn()
+      }
+
+      wrapper = mount(GenerateTestRun, {
+        localVue,
+        mocks: {
+          $router,
+        },
+        data() {
+          return {
+            plateSpecs: [{ numberOfPlates: 1, numberOfPositives: 2 }],
+            addToDart: true,
+          }
+        },
+      })
+
+    })
+    it('when the request is successful', async () => {
```

Comment:
I think the above empty line on line 157 was supposed to be after the `})` on line 158